spyder/config/main.py

```diff
@@ -93,6 +93,8 @@
               'cpu_usage/timeout': 2000,
               'use_custom_margin': True,
               'custom_margin': 0,
+              'use_custom_cursor_blinking': False,
+              'custom_cursor_blinking': 1000,

```

Comment:
@jnsebgosselin, your solution is the only one that occurs to me too.

Since we can't adjust this value in our config system, please remove it from here. It'll be written to `spyder.ini` when the spinbox is created.